index.html

```diff
@@ -0,0 +1,327 @@
+        </div>
+        <div
+          class="see-height d-flex justify-content-center align-items-center"
+        >
+          <button class="d-lg-none bg-join">
+            Join L2 Ministries Conference 2021
+          </button>
+          <p class="under-p d-none d-lg-block">SEE THE WHOLE PROGRAM</p>
+        </div>
+      </section>
+    </div>
+
+    <div class="container">
+      <section class="speakers">
+        <div class="my-5">
+
+       
+        <h3 class="text-center">Featured Speakers</h3>
+        <div class=" underline">
+          <hr >
+        </div>
+      </div>
+        <div class="row">
+          <section class="col-md-6 d-flex f-speaker mb-5">
+            <div class="img">
+
+            </div>
+            <div class="d-flex flex-column  ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+            
+          </section>
+          <section class="col-md-6 d-flex   f-speaker mb-5">
+             <div class="img">
+              
+            </div>
+            <div class="d-flex flex-column  ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+          </section>
+        </div>
+        <div class="row">
+          <section class="col-md-6 d-flex  f-speaker mb-5">
+            <div class="img">
+              
+            </div>
+            <div class="d-flex flex-column ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+          </section>
+          <section class="col-md-6 d-flex  f-speaker mb-5">
+            <div class="img">
+              
+            </div>
+            <div class="d-flex flex-column  ps-3">
+              <h3></h3>
+              <p></p>
+              <p></p>
+            </div>
+          </section>
+        </div>
+      </section>
+    </div>
+    <footer>
+      <div class="container-fluid footer-bg ">
+          <section class="container d-flex flex-column justify-content-center align-items-center">
+              <div class="partner-header d-flex flex-column justify-content-center align-items-center">
+  
+          
+  <h4>Partner</h4>
+  <div class=" underline">
+      <hr >
+    </div>
+   
+  </div>
+  <ul class="d-flex list-unstyled flex-wrap justify-content-around col-12">
+    <li>Petra Ministries</li>
+    <li>Mustard Seed Unilab </li>
+    <li>TACSFON UI</li>
+  </ul>
+       </section>
+      </div>
+      <section class="container-flex d-flex justify-content-around footer-lg align-items-center">
+  
+       
+        <div class="navbar-brand d-none d-md-block"
+        >              <img src="img/icons/idea.png" alt="brand"> 
+        <h1>
+          L2 Global <br> Conference 2021
+        </h1></div>
+          <p class="text-center">2021 Illuminations. some right reserved</p>
+      </section>
+      <p class="text-center">  
+        License and Credits to Cindy Shin: https://www.behance.net/adagio07</p>
+  </footer>
```

Comment:
The footer section is missing a logo on the left

- [ ] Kindly add the logo on the left.
 _see reference below_
![image](https://user-images.githubusercontent.com/67957533/132740736-669eaf39-94f2-4366-9471-ebbcc01ee74c.png)

Reply:
This footer section has a logo already on it, please kindly check this screenshot below
![logo](https://user-images.githubusercontent.com/65824881/132748503-cb104b30-3a70-477c-8b12-82dec2db4260.png)